pkg/hook/config/config_v1.go

```diff
@@ -520,13 +517,12 @@ func convertMutating(cfgV1 KubernetesAdmissionConfigV1) (htypes.MutatingConfig,
 	return cfg, nil
 }
 
-func (cv1 *HookConfigV1) CheckConversion(kubeConfigs []htypes.OnKubernetesEventConfig, cfgV1 KubernetesConversionConfigV1) (allErr error) {
-	var err error
+func (cv1 *HookConfigV1) CheckConversion(kubeConfigs []htypes.OnKubernetesEventConfig, cfgV1 KubernetesConversionConfigV1) error {
+	var allErr error
 
 	if len(cfgV1.IncludeSnapshotsFrom) > 0 {
-		err = CheckIncludeSnapshots(kubeConfigs, cfgV1.IncludeSnapshotsFrom...)
-		if err != nil {
-			allErr = multierror.Append(allErr, fmt.Errorf("includeSnapshotsFrom is invalid: %v", err))
+		if err := CheckIncludeSnapshots(kubeConfigs, cfgV1.IncludeSnapshotsFrom...); err != nil {
+			allErr = multierror.Append(allErr, fmt.Errorf("includeSnapshotsFrom is invalid: %w", err))
 		}
 	}
 
```

Comment:
If i understand it correctly, it will be little short
```suggestion
	if len(cfgV1.IncludeSnapshotsFrom) > 0 {
		if err := CheckIncludeSnapshots(kubeConfigs, cfgV1.IncludeSnapshotsFrom...); err != nil {
			return fmt.Errorf("includeSnapshotsFrom is invalid: %w", err)
		}
	}

    return nil
```

Reply:
IDE makes too strange suggestion. i mean lines 524-529

Reply:
yep, overlooked it